spec/unit/shoulda/matchers/active_record/validate_uniqueness_of_matcher_spec.rb

```diff
@@ -726,6 +726,51 @@
     end
   end
 
+  context 'when the writer method for the attribute changes incoming values' do
+    context 'and ignoring_case_sensitivity is not specified' do
+      it 'raises a CouldNotSetAttributeError' do
+        model = define_model_validating_uniqueness(
+          attribute_name: :name,
+          validation_options: { case_sensitive: false },
+        )
+
+        model.class_eval do
+          def name=(name)
+            super(name.upcase)
+          end
+        end
+
+        assertion = lambda do
+          expect(model.new).
+            to validate_uniqueness_of(:name).
+            case_insensitive
+        end
+
+        expect(&assertion).to raise_error(
+          Shoulda::Matchers::ActiveModel::AllowValueMatcher::CouldNotSetAttributeError
```

Comment:
Line is too long. [86/80]<br>Put a comma after the last parameter of a multiline method call.